dali/operators/generic/pad.h

```diff
@@ -100,7 +100,7 @@ class Pad : public Operator<Backend> {
         }
       }
       auto shape = shape_.tensor_shape_span(i);
-      DALI_ENFORCE(static_cast<int>(axes_.size()) == shape.size(),
+      DALI_ENFORCE(axes_.size() == shape.size(),
```

Comment:
This will result in signed/unsigned comparison warning and will break our build. Please revert.